src/Raven.Studio/typescript/common/queryUtil.ts

```diff
@@ -47,7 +47,7 @@ class queryUtil {
     static formatIndexQuery(indexName: string, fieldName: string, value: string) {
         const escapedFieldName = queryUtil.wrapWithSingleQuotes(fieldName);
         const escapedIndexName = queryUtil.escapeName(indexName);
-        return `from index ${escapedIndexName} where ${escapedFieldName} = '${value}' `;
+        return `from index ${escapedIndexName} where ${escapedFieldName} == '${value}' `;
```

Comment:
I think this is outside of this issue, but could you check if value needs to be escaped? Looks like it is missing?

Reply:
done